python-frontend/src/main/java/org/sonar/python/semantic/v2/types/Assignment.java

```diff
@@ -40,9 +40,9 @@ public Assignment(SymbolV2 lhsSymbol, Name lhsName, Expression rhs, Map<SymbolV2
     this.propagationsByLhs = propagationsByLhs;
   }
 
-  void computeDependencies(Expression expression, Set<SymbolV2> trackedVars) {
+  void computeDependencies(Set<SymbolV2> trackedVars) {
```

Comment:
`computeDependencies(...)` was used by one call side and it always passed `a.rhs()` as the expression. Thus the parameter can be omitted